site/hxapi/transfers/view.md

```diff
@@ -0,0 +1,268 @@
+| ContactNumbers       | Contact information including emergency numbers and reconfirmation details.   |
+
+### Example
+
+{% codetabs %}
+{% codetab XML %}
+```xml
+<Booking>
+  <BookingRef>TBBGNHMT</BookingRef>
+  <SupplierRef>123456</SupplierRef>
+  <Status>Confirmed</Status>
+  <DateBooked>2025-01-15T10:30:00Z</DateBooked>
+  <DateCancelled/>
+  <Email>customer@example.com</Email>
+  <Product>
+    <Price>103.52</Price>
+    <Currency>GBP</Currency>
+    <Category>private_car</Category>
+    <TransferCode>TBHT1169922</TransferCode>
+    <DeparturePointType>Airport</DeparturePointType>
+    <ArrivalPointType>Hotel</ArrivalPointType>
+    <Cancellation>
+      <Period>24</Period>
+      <Percentage>100</Percentage>
+      <Fee>0</Fee>
+    </Cancellation>
+    <VehicleDetails>
+      <MinCapacity>1</MinCapacity>
+      <MaxCapacity>4</MaxCapacity>
+      <MinStops/>
+      <MaxStops/>
+      <Bags>4</Bags>
+      <NumberOfVehicles>1</NumberOfVehicles>
+      <IsPrivate>true</IsPrivate>
+      <Supplier>Transfer Company Ltd</Supplier>
+      <Make>Mercedes</Make>
+    </VehicleDetails>
+    <OutboundTransfer>
+      <JourneyId>OUT123456</JourneyId>
+      <SupplierRef>ABC123</SupplierRef>
+      <Origin>Málaga Airport</Origin>
+      <OriginAddress>Málaga Airport, Málaga</OriginAddress>
+      <OriginIata>AGP</OriginIata>
+      <Destination>Hotel Sol Marbella</Destination>
+      <DestinationAddress>Av. del Mar, 29600, Marbella</DestinationAddress>
+      <DestinationIata/>
+      <JourneyTime>60</JourneyTime>
+      <DepartureDate>2025-08-01</DepartureDate>
+      <DepartureTime>10:00</DepartureTime>
+      <ArrivalDate>2025-08-01</ArrivalDate>
+      <ArrivalTime>11:00</ArrivalTime>
+      <PickupDate>2025-08-01</PickupDate>
+      <PickupTime>10:00</PickupTime>
+      <InformationUrl>https://example.com/transfer-info</InformationUrl>
+      <JoiningInstruction>Meet at Terminal 3 Arrivals Hall</JoiningInstruction>
+      <ContactNumbers>
+        <UkEmergencyTel>+44 20 7123 4567</UkEmergencyTel>
+        <ReconfirmationTel>+34 952 123 456</ReconfirmationTel>
+        <EmergencyTel>+34 952 999 888</EmergencyTel>
+      </ContactNumbers>
+    </OutboundTransfer>
+    <ReturnTransfer>
+      <JourneyId>RET123456</JourneyId>
+      <SupplierRef>DEF456</SupplierRef>
+      <Origin>Hotel Sol Marbella</Origin>
+      <OriginAddress>Av. del Mar, 29600, Marbella</OriginAddress>
+      <OriginIata/>
+      <Destination>Málaga Airport</Destination>
+      <DestinationAddress>Málaga Airport, Málaga</DestinationAddress>
+      <DestinationIata>AGP</DestinationIata>
+      <JourneyTime>60</JourneyTime>
+      <DepartureDate>2025-08-07</DepartureDate>
+      <DepartureTime>12:30</DepartureTime>
+      <ArrivalDate>2025-08-07</ArrivalDate>
+      <ArrivalTime>13:30</ArrivalTime>
+      <PickupDate>2025-08-07</PickupDate>
+      <PickupTime>12:30</PickupTime>
```

Comment:
the pickup time shouldn't be the same as the departure time if one's for the flight and one's for the transfer.

Similar issues for the outbound times too.